storage/debugbar/007124838e96ef988c8469481a41ea24.json

```diff
@@ -0,0 +1 @@
0\u03bcs","stmt_id":null,"connection":"rogue"},{"sql":"select * from `signups` where `signups`.`id` = '1' limit 1","params":{"0":"1","hints":"Use <code>SELECT *<\/code> only if you need all columns from table<br \/><code>LIMIT<\/code> without <code>ORDER BY<\/code> causes non-deterministic results, depending on the query execution plan"},"duration":0.0005,"duration_str":"500\u03bcs","stmt_id":null,"connection":"rogue"}]},"swiftmailer_mails":{"count":0,"mails":[]},"session":[],"request":{"format":"html","content_type":"application\/json","status_text":"OK","status_code":"200","request_query":"array:3 [\n  \"filter\" => array:1 [\n    \"campaign_id\" => \"1283\"\n  ]\n  \"as_user\" => \"5589c991a59dbfa93d8b45ae\"\n  \"limit\" => \"6\"\n]","request_request":"array:3 [\n  \"filter\" => array:1 [\n    \"campaign_id\" => \"1283\"\n  ]\n  \"as_user\" => \"5589c991a59dbfa93d8b45ae\"\n  \"limit\" => \"6\"\n]","request_headers":"array:6 [\n  \"content-type\" => array:1 [\n    0 => \"\"\n  ]\n  \"content-length\" => array:1 [\n    0 => \"\"\n  ]\n  \"x-ds-rogue-api-key\" => array:1 [\n    0 => \"abc123\"\n  ]\n  \"user-agent\" => array:1 [\n    0 => \"GuzzleHttp\/6.2.1 curl\/7.35.0 PHP\/7.0.6-9+donate.sury.org~trusty+2\"\n  ]\n  \"host\" => array:1 [\n    0 => \"rogue.dev:8000\"\n  ]\n  \"x-request-id\" => array:1 [\n    0 => \"1494355063.9736-0\"\n  ]\n]","request_server":"array:29 [\n  \"APP_ENV\" => \"local\"\n  \"USER\" => \"vagrant\"\n  \"HOME\" => \"\/home\/vagrant\"\n  \"FCGI_ROLE\" => \"RESPONDER\"\n  \"QUERY_STRING\" => \"filter%5Bcampaign_id%5D=1283&as_user=5589c991a59dbfa93d8b45ae&limit=6\"\n  \"REQUEST_METHOD\" => \"GET\"\n  \"CONTENT_TYPE\" => \"\"\n  \"CONTENT_LENGTH\" => \"\"\n  \"SCRIPT_FILENAME\" => \"\/home\/vagrant\/DoSomething.org\/rogue\/public\/index.php\"\n  \"SCRIPT_NAME\" => \"\/index.php\"\n  \"REQUEST_URI\" => \"\/api\/v1\/reportbacks?filter%5Bcampaign_id%5D=1283&as_user=5589c991a59dbfa93d8b45ae&limit=6\"\n  \"DOCUMENT_URI\" => \"\/index.php\"\n  \"DOCUMENT_ROOT\" => \"\/home\/vagrant\/DoSomething.org\/rogue\/public\"\n  \"SERVER_PROTOCOL\" => \"HTTP\/1.1\"\n  \"GATEWAY_INTERFACE\" => \"CGI\/1.1\"\n  \"SERVER_SOFTWARE\" => \"nginx\/1.8.0\"\n  \"REMOTE_ADDR\" => \"10.0.2.2\"\n  \"REMOTE_PORT\" => \"60452\"\n  \"SERVER_ADDR\" => \"10.0.2.15\"\n  \"SERVER_PORT\" => \"80\"\n  \"SERVER_NAME\" => \"rogue.dev\"\n  \"REDIRECT_STATUS\" => \"200\"\n  \"HTTP_X_DS_ROGUE_API_KEY\" => \"******\"\n  \"HTTP_USER_AGENT\" => \"GuzzleHttp\/6.2.1 curl\/7.35.0 PHP\/7.0.6-9+donate.sury.org~trusty+2\"\n  \"HTTP_HOST\" => \"rogue.dev:8000\"\n  \"HTTP_X_REQUEST_ID\" => \"1494355063.9736-0\"\n  \"PHP_SELF\" => \"\/index.php\"\n  \"REQUEST_TIME_FLOAT\" => 1494355064.0204\n  \"REQUEST_TIME\" => 1494355064\n]","request_cookies":"[]","response_headers":"array:2 [\n  \"cache-control\" => array:1 [\n    0 => \"no-cache\"\n  ]\n  \"content-type\" => array:1 [\n    0 => \"application\/json\"\n  ]\n]","path_info":"\/api\/v1\/reportbacks","session_attributes":"[]"}}
```

Comment:
oh right! I thought for some reason it got added as extra commits. Do I just put `/storage/debugar` in the `.gitignore` file?